security-analysis/security-analysis-api/src/main/java/com/powsybl/security/action/GeneratorAction.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.security.action;
+
+import java.util.Objects;
+import java.util.Optional;
+
+/**
+ * An action to:
+ * - increase or decrease the targetV of a generator.
+ * - move to voltage control or remove from voltage control.
+ * - change targetQ of a generator that is not a voltage controller.
+ * - change targetV of a generator that is a voltage controller.
```

Comment:
targetV and targetQ can always be changed, independently of voltage control on/off status I think.

voltage control is called voltage regulator in powsybl

```suggestion
 * An action to:
 * - change the targetP of a generator, either by specifying a new absolute value, or a relative change (MW)
 * - enable or disable voltage regulator
 * - change targetQ of a generator (MVAr)
 * - change targetV of a generator (kV)
```

Reply:
Yes of course, but with no impact on a power flow engine. So the issue in the documentation here? Voltage regulator: yes I know but I don't like this choice :-)

Reply:
yes issue here is only documentation